render-utils.js

```diff
@@ -1,53 +1,52 @@
 export function renderDogCard(dog) {
-    const div = document.createElement('div');
-    const a = document.createElement('a');
-    const img = document.createElement('img');
-    const p = document.createElement('p');
+  const div = document.createElement("div");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+  const p = document.createElement("p");
 
-    div.classList.add('dog-card');
+  div.classList.add("dog-card");
 
-    p.textContent = dog.name;
-    img.src = `./assets/${dog.breed}.jpeg`;
-    a.href = '' // link to the dog's detail page here
-    
-    div.append(p, img);
+  p.textContent = dog.name;
+  img.src = `./assets/${dog.breed}.jpeg`;
+  a.href = `/detail/?id=${dog.id}`; // link to the dog's detail page here
 
-    a.append(div);
+  div.append(p, img);
 
-    return a;
+  a.append(div);
+
+  return a;
 }
 
 export function renderDogDetail(dog) {
-    const div = document.createElement('div');
```

Comment:
Looks like you had fe eslint error nice work finding all of them and getting your ci to pass